dvc.yaml

```diff
@@ -0,0 +1,30 @@
+stages:
+  get_data:
+    cmd: python src/get_data.py
```

Comment:
Shouldn't this be `poetry run python src/get_data.py`  to ensure that all Python scripts are executed within the defined Poetry environment? @rkargul